src/Umbraco.Cms.Integrations.Crm.ActiveCampaign/App_Plugins/UmbracoCms.Integrations/Crm/ActiveCampaign/js/configuration.controller.js

```diff
@@ -0,0 +1,16 @@
+﻿function configurationController($scope, notificationsService, umbracoCmsIntegrationsCrmActiveCampaignResource) {
+    var vm = this;
+
+    umbracoCmsIntegrationsCrmActiveCampaignResource.checkApiAccess().then(function (response) {
+
+        if (response.isApiConfigurationValid)
+            vm.account = response.account;
+
+        vm.status = response.isApiConfigurationValid 
+            ? "Connected. Account name: "
```

Comment:
This doesn't look quite right - aren't you missing concatenating the account name?

Reply:
Understood - thanks.